src/lib/middleware/searchMiddleware.tsx

```diff
@@ -0,0 +1,25 @@
+import { getContextPageFromReq } from "lib/getContextPage"
+import { OwnerType } from "@artsy/cohesion"
+import { stringify } from "qs"
+
+export const searchMiddleware = async (req, res, next) => {
+  const { pageType } = getContextPageFromReq(req)
+
+  if (pageType === OwnerType.search) {
```

Comment:
Now that this route is scope to the search page in express, the check should be removed.